server/tournaments/index.js

```diff
@@ -154,6 +177,7 @@ class Tournament {
 	forceEnd() {
 		if (this.isTournamentStarted) {
 			if (this.autoDisqualifyTimer) clearTimeout(this.autoDisqualifyTimer);
+			if (!this.inProgressMatches) throw new Error(`Unexpected null in tours`);
```

Comment:
I don't know if I like all the new `Unexpected null` throws. Especially if they do the same thing as `// @ts-ignore`

Reply:
I've thought about this, and I think the best way is to declare these as non-null (and cover them up with `ts-ignore` or an `any`-cast.